website/docs/r/hyper_converged_cluster.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "AzureStackHCI"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_hyper_converged_cluster"
```

Comment:
this is referred to as azure stack HCI everywhere, so why are we using this resource name?
```suggestion
page_title: "Azure Resource Manager: azurerm_azure_stack_hci_cluster"
```
or
```suggestion
page_title: "Azure Resource Manager: azurerm_hci_cluster"
```

Reply:
I remember you suggest to not define the name with abbreviation in previous PR review. So I updated the name with full name. Based on above options, I suggest to use `azurerm_hci_cluster` since the folder name includes "azurestackhci".

![image](https://user-images.githubusercontent.com/19754191/100034823-fac18e80-2e37-11eb-82a4-e8d36b065a0a.png)

Reply:
Updated

Reply:
Generally yes, but given all the docs use HCI i think it is the more user friendly option, hyper converged cluster is much less likely to be recognizable to a user/not show up in a search